pset-functions.js

```diff
@@ -136,6 +182,32 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+let letters = {
+    a: 1,b: 3,c: 3,d: 2,
+    e: 1,f: 4,g: 2,h: 4,
+    i: 1,j: 8,k: 5,l: 1,
+    m: 3,n: 1,o: 1,p: 3,
+    q: 10,r: 1,s: 1,t: 1,
+    u: 1,v: 4,w: 4,x: 8,
+    y: 4,z: 10
+  }
+
+  score = 0
+
+  function wordScore(word) {
+
+    let wordArray = word.toLowerCase().split("")
+
+    for (let i = 0; i < wordArray.length; i++) {
+          score = score + parseInt(letters[wordArray[i]])
+        }
+
+      console.log(score)
```

Comment:
@rvaswani85 this works, but you should return the score value itself, instead of using `console.log()` to print from within the function. Instead, you should console.log() values "outside of the function". That way, your `wordScore()` function is more flexible.